src/server/app.ts

```diff
@@ -48,6 +51,27 @@ function addRoutes(attach: string, router: Router | Handler): void {
   app.use(SERVER_SETTINGS.getServerRoot() + attach, router);
 }
 
+// Add request metrics
```

Comment:
As this also adds metrics for static files, maybe we can limit it to the `/plywood` endpoint?